internal/kgateway/translator/irtranslator/route.go

```diff
@@ -208,6 +207,11 @@ func (h *httpRouteConfigurationTranslator) envoyRoutes(
 			out.Action = &envoy_config_route_v3.Route_DirectResponse{
 				DirectResponse: &envoy_config_route_v3.DirectResponseAction{
 					Status: http.StatusInternalServerError,
+					Body: &envoy_config_core_v3.DataSource{
+						Specifier: &envoy_config_core_v3.DataSource_InlineString{
+							InlineString: `kgateway detected an invalid route configuration and replaced it with a direct response.`,
```

Comment:
nit: kgateway is implicit
```suggestion
							InlineString: `invalid route configuration detected and replaced with a direct response`,
```